sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -336,6 +339,9 @@
     for (final String tracingOrigin : propertiesProvider.getList("tracing-origins")) {
       options.addTracingOrigin(tracingOrigin);
     }
+    for (final String proguardUuids : propertiesProvider.getList("proguard-uuids")) {
```

Comment:
I'm not sure that an App can have more than 1 proguard-uuid, afaik the final App always output a single mapping file, so it'd not make sense to be a list I guess

Reply:
All examples, including Android one from you has UUIDs represented as an array in #1236. I've never used Proguard.

Reply:
@maciejwalkowiak it's because the `DebugMeta#images` is a list, since it's also used for debug images from native code, but proguard mapping file per see is a single one afaik.

We always add a single proguard id to the images list though.

Reply:
Changed from `proguard-uuids` (plural) to `proguard-uuid` (singular). There is just one string now.